node/src/node_configurator/mod.rs

```diff
@@ -54,13 +54,23 @@ pub fn determine_config_file_path(
     .collect();
     let orientation_vcl = CommandLineVcl::from(orientation_args);
     let multi_config = make_new_multi_config(&orientation_schema, vec![Box::new(orientation_vcl)])?;
-    let config_file_path = value_m!(multi_config, "config-file", PathBuf).expectv("config-file");
+    let config_file_path: PathBuf =
+        value_m!(multi_config, "config-file", PathBuf).expectv("config-file");
```

Comment:
Probably `: PathBuf` is unnecessary here. Also, the `.expectv()` call should be replaced with a standard `expect()` call complaining that the `config-file` parameter is not properly defaulted by `clap`.